source/Classroom/Basics/Security/security_checklist.md

```diff
@@ -1,49 +1,36 @@
 ---
 seo:
   title: Security Checklist
-  description: Security Checklist
-  keywords: password, security, secure, virus, hack, script
+  description: Tips to help keep your account secure.
+  keywords: password, security, secure, virus, hack, script, github
 title: Security Checklist
 weight: 0
 layout: page
-zendesk_id: 200182938
 navigation:
   show: true
 ---
 
-SendGrid doesn’t compromise when it comes to compromised accounts and neither should you. Every day that you put off security steps, your reputation as a business and a sender is at risk. Take a few minutes, go through this checklist and make sure that your systems are secure.
+SendGrid doesn’t compromise when it comes to compromised accounts and neither should you. Every day that you put off security steps, your reputation as a business and a sender is at risk. Use these suggestions below to improve the security of your system.
```

Comment:
~SendGrid doesn’t compromise when it comes to compromised accounts and neither should you. Every day that you put off security steps, your reputation as a business and a sender is at risk. Use these suggestions below to improve the security of your system.~ 

Use these suggestions to improve the security of your system.